app/src/main/java/com/onboarding/parkingsystemkotlin/utils/ConstantUtils.kt

```diff
@@ -0,0 +1,9 @@
+package com.onboarding.parkingsystemkotlin.utils
+
+class ConstantUtils {
```

Comment:
instead of this, you should use an object.

object ConstantUtils {
        const val PARKING_VIEW_TAG: String = "ParkingView"
        const val CONFIGURE_PARKING_LOTS_DIALOG_TAG = "ConfigureParkingDialog"
        const val PARKING_LOT_NOT_SET: Int = -1
}

Reply:
switched to object